geode-core/src/main/java/org/apache/geode/internal/cache/PartitionedRegionStats.java

```diff
@@ -176,8 +176,18 @@
   private static final int prMetaDataSentCountId;
 
   private static final int localMaxMemoryId;
+  static final int bucketClearsId;
+  static final int bucketClearTimeId;
+  static final int bucketClearsInProgressId;
 
   static {
+    final String bucketClearsDesc =
+        "The total number of times a clear has been done on this region and it's bucket regions";
+    final String bucketClearTimeDesc =
+        "The total amount of time spent doing a bucket clear onx bucket regions";
```

Comment:
note the "onx" type. Better would be: "The total amount of time, in nanoseconds, spend clearing buckets on this partitioned region."

Reply:
Done

Reply:
Done